BD-3508 Add note about hourly Segment Extension refreshes (#8137)
braze-inc/braze-docs: _docs/_user_guide/engagement_tools/segments/segment_extension.md

## Step 4: Designate refresh settings (optional)

> If you don't need your segment to refresh on a regular schedule, you can save it without using refresh settings, and Braze will default to generating your segment based on your user membership at that moment. Use the default behavior if you only want to generate a segment once and then target it with one-off campaign.

Reviewer: ```suggestion
If you don't need your extension to refresh on a regular schedule, you can save it without using refresh settings, and Braze will default to generating your Segment Extension based on your user membership at that moment. Use the default behavior if you only want to generate the audience once and then target it with a one-off campaign.
```

### Saving a snapshot in time

> You can specify whether you want this extension to represent a single snapshot in time, or whether you want this extension to refresh on a recurring schedule.

Reviewer: Can we use this section to explain what saving a snapshot in time means rather than repeating the line from the previous section? (I will also say the phrase "snapshot in time" sounds like an idiom. We might want to say something more plain-language.) 

It would also be a good place to provide some strategic guidance—why would they want a snapshot vs refreshing? 

For example, I could see that in industries with strict regulations (like, finance or healthcare), marketers may opt for a snapshot to ensure compliance with data usage policies and to avoid any changes that could lead to unintended targeting.

Reviewer: Probably should also note that this option is the default (they don't technically have to choose this)

### Setting up a recurring refresh

To set up a recurring schedule, select **Refresh Settings** in the upper right corner of your specific extension. The option to designate refresh settings is available for all types of Segment Extensions, including SQL segments, CDI segments, and simple form-based Segment Extensions.

{% alert important %}

> The setting to regenerate extensions daily is automatically turned off for unused Segment Extensions. Braze defines unused extensions as ones that meet the following criteria:

Reviewer: ```suggestion
Refresh settings are automatically turned off for unused Segment Extensions. Braze defines unused extensions as ones that meet the following criteria:
```
